tests/lib/external/snapd-testing-tools/utils/spread-shellcheck

```diff
@@ -119,21 +120,29 @@ def checksection(data, env: Dict[str, str]):
 
     for key, value in env.items():
         value = str(value)
-        # Unpack the special "$(HOST: ...) syntax and tell shellcheck not to
-        # worry about the use of echo to print variable value.
-        if value.startswith("$(HOST:") and value.endswith(")"):
-            script_data.append("# shellcheck disable=SC2116")
-            value = "$({})".format(value[len("$(HOST:"):-1])
-        # XXX: poor man's shell key=value assignment with values in double
-        # quotes so that one value can refer to another value.
-        if '"' in value:
-            value = value.replace('"', '\"')
+        disabled_warnings = set()
```

Comment:
This patch probably needs to go back to the shared `snapd-testing-tools` repository (cc @sergiocazzolato for awareness).

Reply:
I can also put in a separate PR so we can review separately.

Reply:
Opened https://github.com/snapcore/snapd-testing-tools/pull/41